.ci/generate_bear_requirements.py

```diff
@@ -121,64 +115,60 @@ def get_all_requirements(bears):
         )
 
 
-def write_gem_requirements(requirements):
-    gem_dependencies = []
-    template = env.get_template(GEM_REQUIREMENTS_TEMPLATE_FILE)
+def _to_entry(obj, version=None):
+    entry = {}
 
-    for requirement in requirements:
-        gem_dependencies.append(
-            {'name': requirement.package,
-             'version': requirement.version,
-             'required': requirement.require})
+    if version:
+        entry['version'] = version
+    elif obj.version:
+        entry['version'] = obj.version
+    else:
+        return True
 
-    gemfile_string = template.render(
-        gems=gem_dependencies
-        )
+    return entry
+
+
+def get_gem_requirements(requirements):
+    gem_dependencies = {}
+
+    for requirement in requirements:
+        gem_dependencies[requirement.package] = _to_entry(requirement)
 
-    with open(os.path.join(PROJECT_DIR, "Gemfile"), 'w') as file:
-        file.write(gemfile_string)
+    return gem_dependencies
 
 
-def write_npm_requirements(requirements):
+def get_npm_requirements(requirements):
     npm_dependencies = {}
-    template = env.get_template(NPM_REQUIREMENTS_TEMPLATE_FILE)
 
     for requirement in requirements:
-        req_version = requirement.version
         package_name = requirement.package
+        req_version = requirement.version
         if req_version:
-            if req_version[0] in ('<', '>', '~', '='):
-                npm_dependencies[package_name] = req_version
-            else:
-                npm_dependencies[package_name] = "~" + req_version
-        else:
-            npm_dependencies[package_name] = "*"
```

Comment:
Did we miss this in PR?

Reply:
This was from when `package.json` had items without any version specified, and isnt needed any longer.